src/templates/csharp/TestApp.cs

```diff
@@ -0,0 +1,47 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium;
+using System.Drawing;
+
+namespace AppiumTest
+{
+    public class Tests : TestBase
+    {
+        [SetUp]
+        public void BeforeAll() {
+            Assert.AreNotEqual(
+                "Please update value for the KobitonApiKey constant first. See more at README.md file.",
+                "your_kobiton_api_key",
+                Config.KobitonApiKey
+            );
```

Comment:
This assertion is not working (wrong order of parameters). Please use `Assert.That `instead
```suggestion
            Assert.That(
                Config.KobitonApiKey, Is.Not.EqualTo("your_kobiton_api_key"),
                "Please update value for the KobitonApiKey constant first. See more at README.md file."
            );
```